kvrocks.conf

```diff
@@ -721,6 +721,13 @@ rocksdb.max_bytes_for_level_base 268435456
 # Default: 10
 rocksdb.max_bytes_for_level_multiplier 10
 
+# If yes, RocksDB will prefetch some of data asynchronously.
+# RocksDB apply it if reads are sequential and its internal automatic
+# prefetching.
+
+# Deafult yes
+rocksdb.read_options.async_io yes
```

Comment:
Perhaps, it would be better to set the default value to `false` because this feature is experimental in RocksDB.

Reply:
@git-hulk

Reply:
Agree with @torwig, can turn it to false by default and wait for this feature production ready. But we can encourage users to have a try.